### PR TITLE
LuponMedia Bid Adapter: add adapter back to prebid master

### PR DESCRIPTION
**Pull request by adxpremium**
## Type of change

- [ ] Bugfix
- [ x] Feature
- [ X] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x ] Other

## Description of change
Prebid-js v5.X support

### CODE REVIEW

**Comment by adxpremium** [comment]
@ChrisHuie I am not sure how to manually kick off CircleCI tests.. Could you please explain? Thank you in advance.

**Comment by ChrisHuie** [comment]
@adxpremium the problem is from the branch you are following in circleci I think. I could see your tests but it was under an 'adx' account. Just rekicked off tests from our side

**Comment by adxpremium** [comment]
> @adxpremium the problem is from the branch you are following in circleci I think. I could see your tests but it was under an 'adx' account. You are probably following your branch in circleci instead of the main repo. I just tried to rekick off tests and it didn't work. Previously I was able to..

@ChrisHuie now you should be able to kick off tests. We are not able to find what is wrong with CircleCI tests. Could you please take another look? Thanks!

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/3347

File: modules/luponmediaBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+          includebidderkeys: false
+        }
+      }
+    },
+    user: {
+    }
+  }
+
+  let bidFloor;
+  if (typeof bidRequest.getFloor === 'function' && !config.getConfig('disableFloors')) {
```
Comment (ChrisHuie) [inline modules/luponmediaBidAdapter.js:320]: Could you use the 'utils' function for isFN here?

**Comment by ChrisHuie** [comment]
@adxpremium currently master is on 6.1. Here is the legacy branch for 5.20.x (https://github.com/prebid/Prebid.js/tree/5.20.x-legacy) we are maintaining through the first part of next year if you would like this in versions 5 and 6.

**Review by ChrisHuie — APPROVED** [review]

**Comment by patmmccann** [comment]
@adxpremium there are references to digitrust here, please remove

**Pull request merged**